app/_src/kubernetes-ingress-controller/deployment/k4k8s-enterprise.md

```diff
@@ -6,128 +6,181 @@ Kong for Kubernetes Enterprise is an enhanced version of
 the Open-Source Ingress Controller. It includes all
 Enterprise plugins and comes with 24x7 support for worry-free
 production deployment.
-This is available to enterprise customers of Kong, Inc. only.
-
-## Prerequisites
-
-Before we can deploy Kong, we need to satisfy one prerequisite: create a license
-secret.
-
-To create this secret, provision the `kong` namespace first:
-
-```bash
-$ kubectl create namespace kong
-namespace/kong created
-```
+This is available only for enterprise customers of Kong, Inc.
+
+## Before you begin
+
+1. Create the `kong` namespace in the cluster to create a secret.
+
+    ```bash
+    $ kubectl create namespace kong
+    ```
+    The results should look like this:
+    ```text
+    namespace/kong created
+    ```
+1. Create [Kong Enterprise License secret](#kong-enterprise-license-secret)
```

Comment:
Why does this link further down the page? I'd expect it to be inline

Reply:
fixed